### PR TITLE
[15.0][MIG] purchase_open_qty

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
This module allows purchase users to identify what are the purchase orders
that currently have quantities pending to invoice or to receive.

### CODE REVIEW

File: purchase_open_qty/models/purchase_order.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def _search_pending_qty_to_invoice(self, operator, value):
     )
     qty_to_receive = fields.Float(
         compute="_compute_qty_to_receive",
-        search="_search_qty_to_receive",
+        # search="_search_qty_to_receive",
```
Comment (chienandalu) [inline purchase_open_qty/models/purchase_order.py]: Why?

**Comment by chienandalu** [comment]
Can you add https://github.com/OCA/purchase-workflow/pull/1503 ?

File: purchase_open_qty/models/purchase_order.py (hunk 1 above)
Comment (CasVissers-360ERP) [reply purchase_open_qty/models/purchase_order.py]: _search_qty_to_receive is not defined.

Comment (chienandalu) [reply purchase_open_qty/models/purchase_order.py]: The it should be simply removed

**Comment by HviorForgeFlow** [comment]
@JuanyDForgeflow could you attend comments above? Thanks!

**Comment by JuanyDForgeflow** [comment]
> Can you add #1503 ?

I added this in the migration commit

**Comment by JuanyDForgeflow** [comment]
> @JuanyDForgeflow could you attend comments above? Thanks!

Done!

**Review by chienandalu — APPROVED** [review]
Thanks!:+1:

**Comment by TheerayutEncoder** [comment]
Functional Test 👍

**Review by TheerayutEncoder — APPROVED** [review]

**Comment by DhavalTforgeFlow** [comment]
https://github.com/OCA/purchase-workflow/blob/2ae6a826050701ed6c12d9035be14d1eec9fd87b/purchase_open_qty/models/purchase_order.py#L15

I think, We do not need qty_to_invoice field in this module. "qty_to_invoice" field is now standard field in purchase module. Can you @JuanyDForgeflow  please check and remove the declaration and computation of this field?

Standard field : https://github.com/odoo/odoo/blob/b3199e949ae70307e8181ba0d6237871d6d07af7/addons/purchase/models/purchase.py#L982

**Pull request merged**